app/assets/stylesheets/_style.css.scss

```diff
@@ -20,21 +20,53 @@ nav {
     padding: 0 10px;
   }
 
-  .title {
+  .inner-left {
     @include span-columns(1 of 2);
     height: $_nav_height;
     line-height: $_nav_height;
+
+    >div {
+      display: inline-block;
+    }
+  }
+
+  .inner-right {
+    @include span-columns(1 of 2);
+    height: $_nav_height;
+    line-height: $_nav_height;
+    text-align: right;
+
+    >div {
+      display: inline-block;
+    }
+  }
+
+  .title {
     a {
       color: white;
     }
   }
 
+  .control {
+    font-size: 0;
+    margin-left: 20px;
+
+    a {
+      color: $light-gray;
+      display: inline-block;
+      font-size: 1rem;
+      height: $_nav_height;
```

Comment:
Name of variable `nav_height` should be written in all lowercase letters with hyphens instead of underscores